resources/dicts/patrols/general/medcat.json

```diff
@@ -212,5 +212,83 @@
+    "chance_of_success": 60,
+    "exp": 10,
+    "success_text": [
+        "p_l eventually relents and begins teaching r_c more about StarClan and the nature of prophecies. Expecting boredom, p_l was shocked when they saw wonder in r_c's face. The two spend the rest of the patrol pondering the nature of their ancestors together.",
+        "p_l eventually relents and begins teaching r_c more about StarClan and the nature of prophecies. r_c takes to it quickly, marveling at their ancestors. They attempt to bub p_l about any recent prophecies, but p_l only laughs and gets the patrol back on track for herbs.",
+        "p_l had seen before how connected s_c was to StarClan, and began telling them more than they would have told any other Clan cat about their ancestors. p_l and s_c spend much of their patrol gathering herbs and discussing the hard-hitting questions on prophecies and StarClan's gifts."
+    ],
+    "fail_text":[
+        "p_l tries to simplify the mystical power of their ancestors, but it only leaves r_c more confused about StarClan and their prophecies.",
+        null,
+        null,
+        null
+    ],
+    "min_cats": 2,
+    "max_cats": 2,
+    "win_skills": ["strong connection to starclan"],
+    "win_trait": null,
+    "antagonize_text": null,
+    "antagonize_fail_text": null
+},
+{
+    "patrol_id": "gen_med_medwar_natureofdeath",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["med_cat", "warrior", "random_herbs", "no_app"],
+    "intro_text": "To p_l's concern, their warrior escort is unusually quiet. When prompted on their troubles, r_c comments that they could not help but think of what legacy they would leave behind if StarClan calls upon them.",
+    "decline_text": "p_l isn't sure how to exactly address such a statement, and quickly attempts to distract r_c with some herb collecting.",
+    "chance_of_success": 60,
+    "exp": 10,
+    "success_text": [
+        "p_l and r_c spend their time on the patrol discussing how, to StarClan, the end of their life did not mean it was the true end. Even in StarClan, cats had their role to play. r_c's legacy would shine brighter than any star. Comforted by these words, r_c thanks p_l and helps bring back the herbs."
+        null,
+        "p_l knew why this bothered s_c so much. A powerful connection to StarClan for either of them allowed them to understand how their legacy would be told again and again to kits in the future. But p_l did their best to ease s_c's worries. To c_n, s_c would be a legend worth remembering one day. s_c brigtens up at that remark, and the two go back to their herb hunt."
+    ],
+    "fail_text":[
+        "p_l attempts to soothe r_c's worries, but nothing seems to stick. r_c turns back towards the camp, leaving p_l alone to fruitlessly hunt for herbs.",
+        null,
+        null,
+        null
+    ],
+    "min_cats": 2,
+    "max_cats": 2,
+    "win_skills": ["strong connection to starclan"],
+    "win_trait": null,
```

Comment:
can remove this line completely! if any of the skill or trait lists are null/empty, then you can just delete the whole line